lte/gateway/c/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -0,0 +1,408 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "log.h"
+#include "intertask_interface.h"
+#include "directoryd.h"
+#include "conversions.h"
+#ifdef __cplusplus
+}
+#endif
+#include "common_defs.h"
+#include "nas5g_network.h"
+#include "amf_app_state_manager.h"
+
+using namespace std;

```

Comment:
Avoid the `using namespace` directive, instead use the `std::` prefix wherever needed

Reply:
fixed